main-page.css

```diff
@@ -0,0 +1,213 @@
+body {
+    margin: 0;
+}
+.right-pannel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 10px 0 0 10px;
+    width: 80px;
+    height: 250px;
+    background-color: white;
+    position: fixed;
+    float: right;
+    top: 40%;
+    right: 0;
+}
+.icons {
+    width: 50px;
+    height: 50px;
+}
+.icons:first-child {
+    margin-top: 10px;
+}
+.icons:last-child {
+    margin-bottom: 10px;
+}
+
+.main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+}
+.first-screen {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    background-color: rgb(0, 0, 0);
+    color: white;
+}
+.header-part {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 80%;
+    height: 150px;
+    border-bottom: 2px solid white;
+}
+.profile-photo {
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    border: 2px solid white;
+}
+.left-part h1{
+    margin-left: 15px;
+}
+.left-part {
+    display: flex;
+    color: white;
+    margin-left: 30px;
+}
+.nav {
+    display: flex;
+    justify-content: end;
+    list-style-type: none;
+    font-size: 30px;
+    margin-right: 30px;
+    padding: 0;
+}
+.nav li {
+    padding: 10px;
+}
+.nav a {
+    text-decoration: none;
+    color: white;
+}
+.welcome-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 80%;
+    height: calc(100vh - 150px);
+    color: white;
+}
+.welcome-text h1 {
+    margin:0;
+    margin-top: 5px;
+}
+.welcome-text h2 {
+    margin: 0;
+    margin-top: 15px;
+}
+.text-in-block {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 20%;
+}
+.underlied-text {
+    color: violet;
+}
+.portfolio-button {
+    width: 150px;
+    height: 50px;
+    border: none;
+    border-radius: 20px;
+    font-size: 20px;
+    background-color: white;
+    color: black;
+}
+.portfolio-button:hover {
+    background-color: black;
+    color: white;
+}
+
+
+.about-screen {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    background-color: white;
+    color: black;
+}
+.about-header {
+    width: 80%;
+    text-align: center;
+    font-size: 60px;
+    margin-bottom: 10%;
+}
+.text-photo-block {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 10%;
+}
+.text-about {
+    font-size: 30px;
+    margin-left: 15%;
+    max-width: 800px;
```

Comment:
Use the same approach for all blocks. Few strings above u r using relative values.

Reply:
fixed